src/main/java/org/primefaces/component/datatable/DataTableBase.java

```diff
@@ -114,7 +107,8 @@
         escapeText,
         rowEditMode,
         stickyTopAt,
-        globalFilterFunction
+        globalFilterFunction,
+        sortPriority
```

Comment:
Why is sortPriority here on the Datatable looks like you added it to the column?

Reply:
Wrong PR I created a new one, see #6390